docs/operate/provisioning/terraform/index-terraform.mdx

```diff
@@ -0,0 +1,582 @@
+A leading tool in the IaC is HashiCorp Terraform, which supports the major cloud providers and services with its providers and modules cloud infrastructure automation ecosystem for provisioning, compliance, and management of any cloud, infrastructure, and service.
+
+### What is Terraform?
+
+Terraform is an open source IaC software tool that provides a consistent CLI workflow to manage hundreds of cloud services. Terraform codifies cloud APIs into declarative configuration files, which can then be shared amongst team members, treated as code, edited, reviewed, and versioned. It enables you to safely and predictably create, change, and improve infrastructure. 
+
+### Capabilities of Terraform
+
+- Terraform is not just a configuration management tool. It also focuses on the higher-level abstraction of the data center and associated services, while allowing you to use configuration management tools on individual systems.
+- It supports multiple cloud providers, such as AWS, GCP, Azure, DigitalOcean, etc.
+- It provides a single unified syntax, instead of requiring operators to use independent and non-interoperable tools for each platform and service.
+- Manages both existing service providers and custom in-house solutions.
+- Terraform is easily portable to any other provider.
+- Provides immutable infrastructure where configuration changes smoothly.
+- Supports client-only architecture, so no need for additional configuration management on a server.
+- Terraform is very flexible, using a plugin-based model to support providers and provisioners, giving it the ability to support almost any service that exposes APIs.
+- It is not intended to give low-level programmatic access to providers, but instead provides a high-level syntax for describing how cloud resources and services should be created, provisioned, and combined.
+- It provides a simple, unified syntax, allowing almost any resource to be managed without learning new tooling.
+
+### The HashiCorp Terraform Redis Enterprise Cloud provider
+
+Redis has developed a Terraform provider for Redis Enterprise Cloud. The HashiCorp Terraform Redis Enterprise Cloud provider allows customers to deploy and manage Redis Enterprise Cloud subscriptions, databases, and network peering easily as code, on any cloud provider. It is a plugin for Terraform that allows Redis Enterprise Cloud Flexible customers to manage the full life cycle of their subscriptions and related Redis databases.
+The Redis Enterprise Cloud provider is used to interact with the resources supported by Redis Enterprise Cloud. The provider needs to be configured with the proper credentials before it can be used. Use the navigation to the left to read about the available provider resources and data sources.
```

Comment:
```suggestion

The Redis Enterprise Cloud provider is used to interact with the resources supported by Redis Enterprise Cloud. The provider needs to be configured with the proper credentials before it can be used. Use the navigation to the left to read about the available provider resources and data sources.
```